templates/pgbouncer/pgbouncer-networkpolicy.yaml

```diff
@@ -44,19 +44,9 @@ spec:
 {{- if .Values.pgbouncer.extraNetworkPolicies}}
 {{ toYaml .Values.pgbouncer.extraNetworkPolicies | indent 4}}
 {{- end}}
-    {{- if .Values.platform.release }}
-    - namespaceSelector: {}
```

Comment:
How are we applying this one still? I don't see any way to add this back in?

Reply:
Good eye, but right above this @dimberman already added the `extraNetworkPolicies` chunk for KEKA support. I essentially copied this chunk to the other `NetworkPolicies` we had to tweak. So this change is just removing the part that was hard-coded to `platform.release`. We'll just have to move this up to Houston somehwere, so made a note of it in my `values.yaml` I referenced in a comment here.